### PR TITLE
Single video page and like feature

### PR DESCRIPTION
**Pull request by gadgetfather**
- Added single video page 
- user can now like the videos
### test credentials are 
email : 123@gmail.com
password: 123

## Preview 
like

### CODE REVIEW

File: src/context/like-context.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      const response = await axios.post(
+        "/api/user/likes",
+        { video },
+        { headers: { authorization: encodedToken } }
+      );
+      if (response.status === 201) {
+        setLikedArr(response.data.likes);
+      }
+    } catch (err) {
+      console.log(err);
```
Comment (Jay-Gadhiya) [inline src/context/like-context.js:21]: you can put return or alert instead of console

File: src/context/like-context.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+  };
+  const removeFromLike = async (videoId) => {
```
Comment (Jay-Gadhiya) [inline src/context/like-context.js:24]: try to put functions in other files for clean code.

File: src/pages/LikedPage/LikedPage.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  const [likedVideos, setLikedVideos] = useState([]);
+  const getLikedVideos = async () => {
+    try {
+      const encodedToken = localStorage.getItem("Manaplay.encodedToken");
+      const response = await axios.get("/api/user/likes", {
+        headers: { authorization: encodedToken },
+      });
+
+      setLikedVideos(response.data.likes);
+    } catch (error) {
```
Comment (Jay-Gadhiya) [inline src/pages/LikedPage/LikedPage.jsx:15]: you can check for status code as you check in previous request

**Review by Jay-Gadhiya — APPROVED** [review]
- overall code looks good

**Review by Kevin-Solomon — APPROVED** [review]

File: src/pages/LikedPage/LikedPage.jsx (hunk 3 above)
Comment (gadgetfather) [reply src/pages/LikedPage/LikedPage.jsx:15]: yes i will update it

**Pull request merged**